insomnia.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Ideally, this file should be submitted and maintained upstream.

Reply:
I've requested this [in the upstream discussion](https://github.com/Kong/insomnia/issues/1812#issuecomment-849766586).

It'll remove it from here once upstream updates their tarballs.